pkg/console/helper.go

```diff
@@ -0,0 +1,96 @@
+package console
+
+import (
+	"github.com/harvester/harvester-installer/pkg/util"
+	"github.com/harvester/harvester-installer/pkg/widgets"
+	"github.com/jroimartin/gocui"
+)
+
+type passwordWrapper struct {
+	c                *Console
+	passwordV        *widgets.Input
+	passwordConfirmV *widgets.Input
+}
+
+func (p *passwordWrapper) passwordVConfirmKeyBinding(g *gocui.Gui, v *gocui.View) error {
+	password1V, err := p.c.GetElement(passwordPanel)
+	if err != nil {
+		return err
+	}
+	userInputData.Password, err = password1V.GetData()
+	if err != nil {
+		return err
+	}
+	if userInputData.Password == "" {
+		return p.c.setContentByName(validatorPanel, "Password is required")
+	}
+	return showNext(p.c, passwordConfirmPanel)
+}
+
+func (p *passwordWrapper) passwordVEscapeKeyBinding(g *gocui.Gui, v *gocui.View) error {
+	p.passwordV.Close()
+	p.passwordConfirmV.Close()
+	if installModeOnly {
+		if canChoose, err := canChooseDataDisk(); err != nil {
+			return err
+		} else if canChoose {
+			return showDataDiskPage(p.c)
+		} else {
+			return showDiskPage(p.c)
+		}
+	}
+	if err := p.c.setContentByName(notePanel, ""); err != nil {
+		return err
+	}
+	return showNext(p.c, tokenPanel)
+}
+
+func (p *passwordWrapper) passwordConfirmVArrowUpKeyBinding(g *gocui.Gui, v *gocui.View) error {
+	var err error
+	userInputData.PasswordConfirm, err = p.passwordConfirmV.GetData()
+	if err != nil {
+		return err
+	}
+	return showNext(p.c, passwordPanel)
+}
+
+func (p *passwordWrapper) passwordConfirmVKeyEnter(g *gocui.Gui, v *gocui.View) error {
+	var err error
```

Comment:
this is to avoid running into 
```non-name userInputData.Password on left side of :=```

The `userInputData.Password` is not a named variable.

Other option would be something like 
```
  tmpValue, err := function()
  if err != nil {
    return err
  }
  userData.Field = tmpValue
```

Reply:
make sense, thanks for the explanation!